prow/config/config.go

```diff
@@ -115,6 +115,10 @@ type JobConfig struct {
 
 	// DecorateAllJobs determines whether all jobs are decorated by default
 	DecorateAllJobs bool `json:"decorate_all_jobs,omitempty"`
+
+	// ProwIgnored is a well known, unparsed field where non-Prow fields can
+	// be defined without conflicting with unknown field validation.
+	ProwIgnored *json.RawMessage `json:"prow_ignored,omitempty"`
```

Comment:
Nit: I would have named it just `Ignored`.

Reply:
I think the specificity will be helpful in cases where there are multiple tools consuming the file.